packages/fxa-settings/src/components/BentoMenu/index.tsx

```diff
@@ -0,0 +1,137 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { useClickOutsideEffect } from 'fxa-react/lib/hooks';
+import { useEscKeydownEffect } from '../../lib/hooks';
+
+import { ReactComponent as FirefoxIcon } from './logo.svg';
+import { ReactComponent as MonitorIcon } from './monitor.svg';
+import { ReactComponent as PocketIcon } from './pocket.svg';
+import { ReactComponent as DesktopIcon } from './desktop.svg';
+import { ReactComponent as MobileIcon } from './mobile.svg';
+import { ReactComponent as LockwiseIcon } from './lockwise.svg';
+import { ReactComponent as BentoIcon } from './bento.svg';
+import { ReactComponent as CloseIcon } from './close.svg';
+import LinkExternal from 'fxa-react/dist/components/LinkExternal';
+
+export const BentoMenu = () => {
+  const [isRevealed, setRevealed] = useState(false);
+  const toggleRevealed = () => setRevealed(!isRevealed);
+  const closeFn = () => setRevealed(false);
+  const bentoMenuInsideRef = useClickOutsideEffect<HTMLDivElement>(setRevealed);
+  useEscKeydownEffect(setRevealed);
+  const dropDownId = 'drop-down-bento-menu';
+
+  return (
+    <>
+      <div className="relative self-center" ref={bentoMenuInsideRef}>
+        {isRevealed ? (
+          <CloseIcon
+            onClick={closeFn}
+            className="invisible cursor-pointer pb-0 transition-standard w-6 mx-6 focus:outline-none desktop:mx-8 tablet:visible"
+          />
+        ) : (
+          <BentoIcon
+            role="img"
+            onClick={toggleRevealed}
+            data-testid="drop-down-bento-menu-toggle"
+            title="Firefox Bento Menu"
+            aria-expanded={isRevealed}
+            aria-controls={dropDownId}
+            className="cursor-pointer p-1 pb-0 focus:outline-none transition-standard w-6 mx-6 desktop:mx-8"
+          />
+        )}
+
+        {isRevealed && (
+          <div
+            id={dropDownId}
+            data-testid={dropDownId}
+            className="rounded tablet:drop-down-menu tablet:top-10 tablet:-left-54"
```

Comment:
The caret looks a little off to me at `-left-54` which is `-13.5rem`:

<img width="384" alt="image" src="https://user-images.githubusercontent.com/13018240/94745873-3280ee00-0341-11eb-829a-17ce29282987.png">


`-12.25rem` seems to center it better:

<img width="362" alt="image" src="https://user-images.githubusercontent.com/13018240/94745980-6825d700-0341-11eb-81b3-0cf3a44fb61c.png">

Reply:
Cleaned this up and got looked exact